application/TFLite_detection_stream.py

```diff
@@ -0,0 +1,181 @@
+# Video Stream Object Detection Using Tensorflow-trained Classifier
+#
+# This program uses a TensorFlow Lite model to perform object detection on a live video stream.
+# To improve FPS, the webcam object runs in a separate thread from the main program.
+# This script will work with codecs supported by CV2 (e.g. MJPEG, RTSP, ...).
+
+import numpy as np
+import cv2
+import time
+import threading
+import datetime
+
+from application.services import sms_service
+
+# Global variables
+min_conf_threshold = 0
+resW, resH = 0, 0
+imW, imH = 0, 0
+FEED_URL = ""
+CWD_PATH = ""
+floating_model = None
+interpreter = None
+input_details = None 
+output_details = None
+labels = None
+height = None
+width = None
+input_mean = 0 
+input_std = 0 
+boxes_idx = 0 
+classes_idx = 0
+scores_idx = 0
+
```

Comment:
Technically if these are global variables, they should be uppercase. We can make a new story for this